_docs/quick-start.md

```diff
@@ -110,21 +131,24 @@ roblox-ts requires [Rojo 0.5.x](https://rojo.space/docs/latest/)
 		}
 	}
 }
-
 ```
-You should add more partitions for the sub-folders of your `out` folder.
+You should add more partitions for the sub-folders of your `out` folder. For more information, please read [Rojo's documentation](https://rojo.space/docs/latest/guide/new-game/).
+{:.info}
 
 The `include` folder should be placed in a shared container that can be accessed from anywhere like `ReplicatedStorage`.
 
-6\. Run `npm install @rbxts/types` to install the Roblox API type definitions.
+6. Add the folders `src`, `out` and `include` in your workspace.
```

Comment:
`out` and `include` won't exist before the user compiles!

Reply:
You're correct, albeit we have some quirks here;

- If the `src` folder doesn't have a single `.ts` file, `out` won't be created;
- If `out` doesn't exist, Rojo will then proceed to complain.

Seemingly since `src` comes with a template file, this shouldn't be a problem once the `init` issue gets resolved.

I'll tackle this on this next commit.